app/code/Magento/Email/view/adminhtml/web/css/source/_module.less

```diff
@@ -0,0 +1,16 @@
+// /**
+//  * Copyright © Magento, Inc. All rights reserved.
+//  * See COPYING.txt for license details.
+//  */
+
+//
+//  Common
+//  _____________________________________________
+
+& when (@media-common =true) {
+    .cms-revision-preview {
+        iframe {
+            height: 98vh;
```

Comment:
Thank you for your contribution. Could you investigate what caused the issue? Or it was always like this? Also need to add fullback for IE 11 or redo the fix.